utils/web3client.py

```diff
@@ -293,6 +293,7 @@ def send_scheduled_transaction(
     def send_all_scheduled_transactions(self, raw_transactions: tp.List[ScheduledTransaction]):
         for trx in raw_transactions:
             self.send_scheduled_transaction(trx)
+            self.wait_for_transaction_receipt(trx.hash())
```

Comment:
please remove it 
It is important for some tests to send all transactions and then wait all transaction but now you do it one by one